### PR TITLE
Add pod annotations

### PR DESCRIPTION
**Pull request by botzill**
Added option to set pod annotations.

### CODE REVIEW

**Comment by botzill** [comment]
@weeco check this, please.

File: kube-eagle/templates/deployment.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ spec:
   template:
     metadata:
       annotations:
+{{- with .Values.podAnnotations }}
+{{- toYaml . | nindent 8 }}
```
Comment (weeco) [inline kube-eagle/templates/deployment.yaml]: Since you are using `nindent` you could indent it correctly for readability.

**Review by weeco — CHANGES_REQUESTED** [review]
Please remove the packaged chart, as I'd like to package it myself to ensure integrity.

I left one more review comment on the nindentation, other than that it looks good to me. Thanks for submitting that PR so quickly :-).

File: kube-eagle/templates/deployment.yaml (hunk 1 above)
Comment (botzill) [reply kube-eagle/templates/deployment.yaml]: Done

**Comment by botzill** [comment]
OK, removed the tar. Check now.

**Comment by weeco** [comment]
Perfect, thank you. Merged and will be packaged within the next few minutes.

**Comment by botzill** [comment]
@weeco Thx!

🎸

**Pull request merged**